model/lifecycle.go

```diff
@@ -537,6 +537,9 @@ func addTasksToBuild(ctx context.Context, creationInfo TaskCreationInfo) (*build
 			continue
 		}
 		bvtu := creationInfo.Project.FindTaskForVariant(t.DisplayName, creationInfo.Build.BuildVariant)
+		if bvtu.HasCheckRun() {
```

Comment:
I think most of the other fields in task is copied from the bvtu here https://github.com/evergreen-ci/evergreen/blob/92a6e0255c64ef8a879aad4c1a7f2661e67cf7bf/model/lifecycle.go#L1162

is there a reason that we should add it here?

Reply:
nope! good catch, thanks! moved